python-client/giskard/core/model_validation.py

```diff
@@ -191,12 +191,16 @@ def validate_label_with_target(classification_labels, target_values=None, target
                 'to make results more understandable in Giskard."'
             )
 
+        _to_append = ""
```

Comment:
_to_append is already private to the method, so no real need to add `_` in front of the name